initialise_server.sql

```diff
@@ -39,54 +39,62 @@ CREATE TABLE IF NOT EXISTS miscellaneous (
-       (34, 320, 'property', 28, 200, 150, 450, 1000, 1200, 1400),
-       (37, 350, 'property', 35, 200, 175, 500, 1100, 1300, 1500),
-       (39, 400, 'property', 50, 200, 200, 600, 1400, 1700, 2000);
-
 CREATE TABLE IF NOT EXISTS properties (
     player_id int UNSIGNED NOT NULL DEFAULT '',
-    game_id int UNSIGNED NOT NULL,
-    state ENUM('unowned', 'owned') NOT NULL DEFAULT 'unowned',
-    property_position tinyint UNSIGNED NOT NULL,
-    house_count tinyint UNSIGNED DEFAULT 0,
-    hotel_count tinyint UNSIGNED DEFAULT 0,
-    FOREIGN KEY (player_id) REFERENCES players(id),
+	game_id int UNSIGNED NOT NULL,
+	state ENUM('unowned', 'owned') NOT NULL DEFAULT 'unowned',
+	mortgaged ENUM('unmortgaged', 'mortgaged') NOT NULL DEFAULT 'unmortgaged',
+	property_position tinyint UNSIGNED NOT NULL,
+	house_count tinyint UNSIGNED DEFAULT 0,
+	hotel_count tinyint UNSIGNED DEFAULT 0,
+	FOREIGN KEY (player_id) REFERENCES players(id),
     FOREIGN KEY (game_id) REFERENCES games(id),
-    FOREIGN KEY (property_position) REFERENCES property_values(property_position)
+	FOREIGN KEY (property_position) REFERENCES property_values(property_position)
 );
 
+INSERT INTO property_values
+VALUES (1, 'Old Kent Road', 60, 'property', 2, 50, 10, 30, 90, 160, 250),
+       (3, 'Whitechapel Road', 60, 'property', 4, 50, 20, 60, 180, 320, 540),
+	   (6, 'The Angel, Islington', 100, 'property', 6, 50, 30, 90, 270, 400, 550),
+	   (8, 'Euston Road', 100, 'property', 6, 50, 30, 90, 270, 400, 550),
+	   (9, 'Pentonville Road', 120, 'property', 8, 50, 40, 100, 300, 450, 600),
+	   (11, 'Pall Mall', 140, 'property', 10, 100, 50, 150, 450, 625, 750),
+	   (13, 'Whitehall', 140, 'property', 10, 100, 50, 150, 450, 625, 750),
+	   (14, 'Northumberland Road', 160, 'property', 12, 100, 60, 180, 500, 700, 900),
+	   (16, 'Bow Street', 180, 'property', 14, 100, 70, 200, 550, 750, 950),
+	   (18, 'Marlborough Street', 180, 'property', 14, 100, 70, 200, 550, 750, 950),
+	   (19, 'Vine Street', 200, 'property', 16, 100, 80, 220, 600, 800, 1000),
+	   (21, 'Strand', 220, 'property', 18, 150, 90, 250, 700, 875, 1050),
+	   (23, 'Fleet Street', 220, 'property', 18, 150, 90, 250, 700, 875, 1050),
+	   (24, 'Trafalgar Square', 240, 'property', 20, 150, 100, 300, 750, 925, 1100),
+	   (26, 'Leicester Square', 260, 'property', 22, 150, 110, 330, 800, 975, 1150),
+	   (27, 'Coventry Street', 260, 'property', 22, 150, 110, 330, 800, 975, 1150),
+	   (29, 'Piccadilly', 280, 'property', 24, 150, 120, 360, 850, 1025, 1200),
+	   (31, 'Regent Street', 300, 'property', 26, 200, 130, 390, 900, 1100, 1275),
+	   (32, 'Oxford Street', 300, 'property', 26, 200, 130, 390, 900, 1100, 1275),
+	   (34, 'Bond Street', 320, 'property', 28, 200, 150, 450, 1000, 1200, 1400),
+	   (37, 'Park Lane', 350, 'property', 35, 200, 175, 500, 1100, 1300, 1500),
+	   (39, 'Mayfair', 400, 'property', 50, 200, 200, 600, 1400, 1700, 2000),
+	   (5, 'Kings Cross Station', 200, 'railroad', 25),
```

Comment:
The railroads and utilities are causing Column count doesn't match errors. This is why the database is empty I think.